docs/content/docs/connectors/datastream/opensearch.md

```diff
@@ -0,0 +1,328 @@
+---
+title: Opensearch
+weight: 5
+type: docs
+aliases:
+  - /dev/connectors/opensearch.html
+  - /apis/streaming/connectors/opensearch.html
```

Comment:
you won't need these since the page didn't exist in the old structures :)

Reply:
Gone, thanks